emperor/core.py

```diff
@@ -278,6 +291,31 @@ def make_emperor(self, standalone=False):
         # avoid unicode strings
         coord_ids = list(map(str, self.mf.index.tolist()))
 
+        # TODO: This will be removed once the custom axes creation is moved to
+        # the graphical user interface i.e. to the Axes tab.
+        if custom_axes:
```

Comment:
I don't think we ever limited the number of custom axes that we have. Is there a reason why we should do that? I may be forgetting a technical limitation.